app/errors/authentication/errors.rb

```diff
@@ -1,10 +1,6 @@
 module Authentication
   module Errors
-    PREVIOUSLY_SUSPENDED_MESSAGE = "It appears that your previous %<community_name>s " \
-                                   "account was suspended. As such, we've taken measures to prevent you from " \
-                                   "creating a new account with %<community_name>s and its community. If you " \
-                                   "think that there has been a mistake, please email us at %<community_email>s, " \
-                                   "and we will take another look.".freeze
+    PREVIOUSLY_SUSPENDED_MESSAGE = "".freeze
```

Comment:
This surprises me a bit, why keep this constant around?

Reply:
No, good catch 😂